### PR TITLE
Update all feedbacks page for Keep Working

### PR DESCRIPTION
**Pull request by maureensturgeon**
- Adds review state: keep working, completed or awaiting review to the feedback entry in the list 
- Moves reviewStates into a different directory (this clutters the PR a bit, if it's distracting I can move to a separate PR)

OPEN QUESTION:
- How do we handle changes to serialization for our APIs? Should I check all individual values or is there a way to confirm that the shape of the response is right?

Screen Shot 2021-06-28 at 11 40 16 AM

## Links

- jira ticket: [LP-1903](https://codedotorg.atlassian.net/browse/LP-1903)

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Review by jamescodeorg — COMMENTED** [review]
Just want to confirm: When there are multiple feedbacks for a level, do we want want to show the "keep working" badge on each of those feedbacks or only the latest one?

File: dashboard/app/controllers/teacher_feedbacks_controller.rb [hunk 1]
```diff
@@ -20,8 +20,24 @@ def index
...
     @feedbacks_as_student_with_level_info = @feedbacks_as_student.map do |feedback|
-      feedback.attributes.merge(feedback&.get_script_level&.summary_for_feedback)
+      level_feedback_key = [feedback.level_id, feedback.script_id].join('_')
+      is_latest = !marked_latest_for_level[level_feedback_key]
+
+      if is_latest
+        marked_latest_for_level[level_feedback_key] = true
+      end
+
+      feedback.summarize.merge({is_latest_for_level: is_latest}).merge(feedback&.get_script_level&.summary_for_feedback)
```
Comment (jamescodeorg) [inline dashboard/app/controllers/teacher_feedbacks_controller.rb:40]: (minor) This line is a little hard to parse, maybe split it up into a few lines?  (No need to do this if you aren't planning on making other changes.)

**Review by jamescodeorg — APPROVED** [review]
Looks good!

File: dashboard/app/controllers/teacher_feedbacks_controller.rb (hunk 1 above)
Comment (maureensturgeon) [reply dashboard/app/controllers/teacher_feedbacks_controller.rb:40]: I'll make a note and circle back on this (don't want to wait another hour for drone)
